plugins/jobs/src/js/components/JobsFormModal.tsx

```diff
@@ -66,6 +66,7 @@ interface JobFormModalState {
   isConfirmOpen: boolean;
 }
 
+const dataLayer: DataLayer = container.get<DataLayer>(DataLayerType);
```

Comment:
Hmm, I see your point, I am wondering if we can get completely rid of them by just putting it into the React Context :) 

Would you block the PR because of this or is it more of a nit-pick concern (I rebased this really a lot and we got some new features that could have already used this that now need refactoring, that's why I'm more on the pragmatic side here)

Reply:
I wouldn't block for this as we can improve forward, but I wanted to discuss it. Would also like to get @nLight thoughts on this.